generator/templates/scripts/TireStatus.js

```diff
@@ -0,0 +1,195 @@
+}
+
+/**
+ * Set the RightRear
+ * @since SmartDeviceLink 2.0.0
+ * @param {SingleTireStatus} rear - The status of the right rear tire. - The desired RightRear.
+ * @returns {TireStatus} - The class instance for method chaining.
+ */
+setRightRear (rear) {
+    this._validateType(SingleTireStatus, rear);
+    this.setParameter(TireStatus.KEY_RIGHT_REAR, rear);
+    return this;
+}
+
+/**
+ * Get the RightRear
+ * @returns {SingleTireStatus} - the KEY_RIGHT_REAR value
+ */
+getRightRear () {
+    let tireStatus = this.getObject(SingleTireStatus, TireStatus.KEY_RIGHT_REAR);
+    if (tireStatus === null) {
+        tireStatus = new SingleTireStatus({
+            status: 'UNKNOWN',
+        });
+        this.setParameter(TireStatus.KEY_RIGHT_REAR, tireStatus);
+        console.warn('TireStatus\'s rightRear was null and will have its status set to UNKNOWN. In the future, this will change to be nullable.');
+    }
+    return tireStatus;
+}
+
+/**
+ * Set the InnerLeftRear
+ * @since SmartDeviceLink 2.0.0
+ * @param {SingleTireStatus} rear - The status of the inner left rear. - The desired InnerLeftRear.
+ * @returns {TireStatus} - The class instance for method chaining.
+ */
+setInnerLeftRear (rear) {
+    this._validateType(SingleTireStatus, rear);
+    this.setParameter(TireStatus.KEY_INNER_LEFT_REAR, rear);
+    return this;
+}
+
+/**
+ * Get the InnerLeftRear
+ * @returns {SingleTireStatus} - the KEY_INNER_LEFT_REAR value
+ */
+getInnerLeftRear () {
+    let tireStatus = this.getObject(SingleTireStatus, TireStatus.KEY_INNER_LEFT_REAR);
+    if (tireStatus === null) {
+        tireStatus = new SingleTireStatus({
+            status: 'UNKNOWN',
+        });
+        this.setParameter(TireStatus.KEY_INNER_LEFT_REAR, tireStatus);
+        console.warn('TireStatus\'s innerLeftRear was null and will have its status set to UNKNOWN. In the future, this will change to be nullable.');
+    }
+    return tireStatus;
+}
+
+/**
+ * Set the InnerRightRear
+ * @since SmartDeviceLink 2.0.0
+ * @param {SingleTireStatus} rear - The status of the inner right rear. - The desired InnerRightRear.
+ * @returns {TireStatus} - The class instance for method chaining.
+ */
+setInnerRightRear (rear) {
+    this._validateType(SingleTireStatus, rear);
+    this.setParameter(TireStatus.KEY_INNER_RIGHT_REAR, rear);
+    return this;
+}
+
+/**
+ * Get the InnerRightRear
+ * @returns {SingleTireStatus} - the KEY_INNER_RIGHT_REAR value
+ */
+getInnerRightRear () {
+    let tireStatus = this.getObject(SingleTireStatus, TireStatus.KEY_INNER_RIGHT_REAR);
+    if (tireStatus === null) {
+        tireStatus = new SingleTireStatus({
+            status: 'UNKNOWN',
+        });
+        this.setParameter(TireStatus.KEY_INNER_RIGHT_REAR, tireStatus);
+        console.warn('TireStatus\'s innerRightRear was null and will have its status set to UNKNOWN. In the future, this will change to be nullable.');
+    }
+    return tireStatus;
+}
+
+// ----------------- END -----------------------
```

Comment:
Dangling END line comment. This can be removed.

Reply:
Removed (d956c3c)